authsites_test.go

```diff
@@ -15,13 +15,13 @@ func Test_AuthSitesDecode(t *testing.T) {
 	// Test empty string
 	err := got.Decode(input)
 	assert.NoError(err)
-	assertEqual(want, got)
+	assert.NoError(assertEqual(want, got))
```

Comment:
Interesting. Is assertEqual likely to throw an error when got is something unexpected?

Reply:
`assertEqual` returns an error if `want` and `got` are not equal. This check and others like it weren't actually working. I don't know why the assert library is set up the way it is. Rather than rework everything, I just wanted to make sure I got it working.